### ISSUE TITLE
[Bug]: Cannot read properties of undefined (reading 'scrollTop') & etc. when connected with a game controller

### ISSUE THREAD

**Issue by gsxgmpkt9020** [ollm/OpenComic#253]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/ollm/OpenComic/issues) for a bug report that matches the one I want to file, without success.

### OpenComic Version

1.2.0master.cc1ebbc & below

### Operating System

WIN 10 22H2 19045.3324

### Steps to reproduce

Open the cbz file

### Expected Behavior

Open it normally without error message.

### Actual Behavior

Open it normally with an error message.

### Error message

```

C:\Program Files\OpenComic\resources\app.asar\scripts\reading.js:4725 Uncaught TypeError: Cannot read properties of undefined (reading 'scrollTop')
    at Object.reading (C:\Program Files\OpenComic\resources\app.asar\scripts\reading.js:4725:28)
    at gamepadLoop (C:\Program Files\OpenComic\resources\app.asar\scripts\gamepad.js:150:19)
C:\Program Files\OpenComic\resources\app.asar\scripts\reading.js:304 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'getBoundingClientRect')
    at disposeImages (C:\Program Files\OpenComic\resources\app.asar\scripts\reading.js:304:61)
    at resized (C:\Program Files\OpenComic\resources\app.asar\scripts\reading.js:2678:4)
C:\Program Files\OpenComic\resources\app.asar\scripts\file-manager.js:2008 extract7z: D:\_CBZ\[單行本／XXX]XXXX.cbz: 3501.384033203125 ms

```

### Additional Information

I have a game controller connected on PC.

[Gamepad.connected](https://developer.mozilla.org/en-US/docs/Web/API/Gamepad/connected) Read only
A boolean indicating whether the gamepad is still connected to the system.

**Comment by ollm** [comment]
I'm not entirely sure, but it seems that both errors are caused by triggering a scroll event while the file is still loading, possibly by the gamepad.

I've made the following change that should ignore these events while the file is still loading 4c93a724220fc820347464a988727ef4649052fe

Build: https://mega.nz/file/DDh2wJaR#pUhJMXeSUjJd99uTz6IDXI_fV5dhfl-0TKoela5IzoQ

**Comment by ollm** [comment]
Try this fix 078ec8ddbf98e4022a43fdb3e5581cd25c707ce4

Build: https://mega.nz/file/bXIVDIoA#CD5cbSrvd_3cON-oq0vhWSxJ3KVqkxfRtZaUl76KeJY
